common/src/main/java/com/microsoft/identity/common/internal/util/BrokerProtocolVersionUtil.java

```diff
@@ -42,4 +43,15 @@ public static boolean canCompressBrokerPayloads(@Nullable String negotiatedBroke
                 MSAL_TO_BROKER_PROTOCOL_COMPRESSION_CHANGES_MINIMUM_VERSION);
 
     }
+
+    public static boolean canFociAppsConstructAccountsFromPrtIdTokens(@Nullable String negotiatedBrokerProtocol) {
```

Comment:
Can we add a test for this.

Reply:
added common4j/src/test/com/microsoft/identity/common/java/util/BrokerProtocolVersionUtilTest.java